docker/README.md

```diff
@@ -42,3 +42,15 @@ example -p 8333:1234 will map the port 8333 internal to the docker container to
 
 ## To connect to the container in a bash shell
 `docker exec -it <container name> bash`
+
+
+
+
+
+
```

Comment:
nit: too much spaces? (once rendered they'll go away, still)

Reply:
idk man, does that really matter? =p

Reply:
it doesn't :smile:

this why I label it as a nitpick and why I approved the PR regardless of my comment.